tests/common/cts_selector.h

```diff
@@ -98,7 +98,7 @@ class cts_selector : public cl::sycl::device_selector {
     }
 
     cl::sycl::string_class vendor =
-        dev.get_platform().get_info<cl::sycl::info::platform::name>();
+        dev.get_platform().get_info<cl::sycl::info::platform::vendor>();
```

Comment:
This looks like a bug fix...

Reply:
Rebooting my brain... Yes as said in the PR description this fixes https://github.com/KhronosGroup/SYCL-CTS/issues/13